controllers/manage.py

```diff
@@ -528,11 +528,13 @@ def SHOW_EMAILS():
         #we can't use the paypal emails etc because we haven't been paid!
         if s.e_mail:
             details = emails(s.OTT_ID, s.name, cnames.get(s.OTT_ID), s.user_sponsor_name, s.e_mail, sponsor_for=s.user_sponsor_kind=='for', email_type='no_payment')
-            if s.user_preferred_image_src and s.user_preferred_image_src_id:
+            if s.user_preferred_image_src and s.user_preferred_image_src_id and local_pic is not None:
```

Comment:
Nothing is setting a ``local_pic`` variable as far as I can see? Seems broken.

Same for the other 2 instances below.

Reply:
It's set in db.py (hacky!)